extensions.json

```
@@ -108,7 +108,8 @@
       "repository": "https://github.com/yarnpkg/berry",
       "version": "2.2.4",
       "checkout": "vscode-zipfs/2.2.4",
-      "location": "packages/vscode-zipfs"
+      "location": "packages/vscode-zipfs",
+      "prepublish": "sed -ri \"s_git\\+https://_https://_g\" package.json"
```

Hmm, looking back at the auto-publishing error, it says:

```
Error: Invalid URL in field 'repository': ssh://git@github.com/yarnpkg/berry.git
```

So I don't think this has to do with `git+https://` vs `https://`, but about the `ssh://git@github.com`.

Instead, maybe it's better to replace the entire offending URL with an acceptable variant (fewer risks of accidental side effects when the pattern is more specific):

```suggestion
      "prepublish": "sed -ri \"s_ssh://git@github.com/yarnpkg/berry.git_https://github.com/yarnpkg/berry_g\" package.json"
```

---

lgtm,